Package.swift

```diff
@@ -1,6 +1,6 @@
 //
```

Comment:
I think we need to make a lot more changes to this file actually.  See https://github.com/JohnSundell/Files/blob/master/Package.swift as an example.

We need to Specify the swift tools version, products, target, test target, etc.

We should probably update the .travis.yml to verify that it builds with `swift build` and tests that way too.

Reply:
Good call. I'll take a look.